src/libs/fileDownload/FileUtils.js

```diff
@@ -116,11 +116,22 @@ function splitExtensionFromFileName(fullFileName) {
     return {fileName: splitFileName.join('.'), fileExtension};
 }
 
+/**
+ * Returns the filename replacing special characters with underscore
+ *
+ * @param {String} fileName
+ * @returns {String}
+ */
+function cleanFileName(fileName) {
+    return fileName.replace(/[^a-zA-Z0-9\-\\._]+/g, '_');
```

Comment:
Quick comment before going through the full review, right now this has an extra \ before the . so wouldn't catch the right characters. Let's also remove the `+` here as noted in https://github.com/Expensify/App/issues/14266#issuecomment-1402264284
```suggestion
    return fileName.replace(/[^a-zA-Z0-9\-\._]/g, '_');
```

Reply:
@dangrous I added `\\` to ignore the `\` as well. But yeah it makes sense to remove that. I've updated the regex. Can you check now?